microsoft-edge/webview2/reference/javascript/hostobjectsyncproxy.yml

```diff
@@ -0,0 +1,145 @@
+### YamlMime:TSType
+name: HostObjectSyncProxy
+uid: 'WebView2Script!HostObjectSyncProxy:class'
+package: WebView2Script!
+fullName: HostObjectSyncProxy
+summary: >-
+  A synchronous host object proxy. Host objects added via `CoreWebView2.AddHostObjectToScript` are exposed as host
+  object proxies using `window.chrome.webview.hostObjects.{name}`<!-- -->. Host object proxies represent a host object.

```

Comment:
```suggestion
  object proxies using `window.chrome.webview.hostObjects.{name}`<!-- -->. A host object proxy represents a host object.
```
Prefer singular, to prevent 1-many ambiguities.

Reply:
@david-risney Would the updated statement still be accurate?

Reply:
Yes looks good.